packages/core/types/src/modules-sdk/__tests__/query.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expectTypeOf } from "expect-type"
+import { FixtureEntryPoints } from "../__fixtures__/remote-query"
+import { QueryGraphFunction } from "../remote-query"
+
+describe("Query", () => {
+  describe("Infer via queryConfig", () => {
+    it("should infer return type of a known entry", async () => {
+      const graph = (() => {}) as unknown as QueryGraphFunction
+      const result = await graph({
+        entryPoint: "product",
+        fields: ["handle", "id"],
+      })
+
+      expectTypeOf(result).toEqualTypeOf<{
+        rows: FixtureEntryPoints["product"][]
```

Comment:
Think rows is an awkward name when we are this many abstraction layers from the database. The more generic data would IMO be better.

Reply:
Any suggestions for the naming?